neuralmonkey/decoders/autoregressive.py

```diff
@@ -475,3 +476,21 @@ def feed_dict(self, dataset: Dataset, train: bool = False) -> FeedDict:
             fd[self.train_mask] = weights
 
         return fd
+
+    @tensor
+    def temporal_states(self) -> tf.Tensor:
+        # strip the last symbol which is </s>
+        return tf.cond(
```

Comment:
Tohle takhle nefunguje. `tf.cond` potřebuje všechny placeholdery, i když se nakonec vyhodnotí opačně.

Reply:
Jo, to už jsem jednou věděl. Myslím, že se to mělo spravit v nějaké další verzi TF. Jak jsme na tom s kompatibilitou s novými verzemi?